docs/sources/get-started/quick-start.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Quick start to run Loki locally
+menuTitle: Loki quick start
+weight: 550
+description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+---
+
+# Quick start to run Loki locally
+
+If you want to experiment with Loki, you can run Loki locally using the Docker Compose file that ships with Loki. It runs Loki in a [scalable monolithic deployment](https://grafana.com/docs/loki/latest/get-started/deployment-modes/#simple-scalable) mode and includes a sample application to generate logs.
```

Comment:
Is that URL correct? Doesn't it point to the SSD version instead of the monolithic one?